src/EnergyPlus/StandardRatings.cc

```diff
@@ -466,27 +466,8 @@ namespace StandardRatings {
 
         // IPLV calculations:
         for (RedCapNum = 0; RedCapNum < NumOfReducedCap; ++RedCapNum) {
-            if (CondenserType == DataPlant::CondenserType::WaterCooled) {
-                // get the entering water temperature for the reduced capacity test conditions
-                if (ReducedPLR[RedCapNum] > 0.50) {
-                    EnteringWaterTempReduced = 8.0 + 22.0 * ReducedPLR[RedCapNum];
-                } else {
-                    EnteringWaterTempReduced = 19.0;
-                }
-                CondenserInletTemp = EnteringWaterTempReduced;
-            } else if (CondenserType == DataPlant::CondenserType::AirCooled) {
-                // get the outdoor air dry bulb temperature for the reduced capacity test conditions
-                if (ReducedPLR[RedCapNum] > 0.3125) {
-                    EnteringAirDryBulbTempReduced = 3.0 + 32.0 * ReducedPLR[RedCapNum];
-                } else {
-                    EnteringAirDryBulbTempReduced = 13.0;
-                }
-                CondenserInletTemp = EnteringAirDryBulbTempReduced;
-            } else { // EvaporativelyCooled Condenser
-                // get the outdoor air wet bulb temperature for the reduced capacity test conditions
-                EnteringAirWetBulbTempReduced = 10.0 + 14.0 * ReducedPLR[RedCapNum];
-                CondenserInletTemp = EnteringAirWetBulbTempReduced;
-            }
+
+            CondenserInletTemp = CondenserEnteringFluidTemperature(state, CondenserType, ReducedPLR[RedCapNum]);
 
```

Comment:
Refactored condenser entering fluid temperature calculations. Also the new function uses corrected equation coefficients to calculate the temperatures.